Hashtable.h

```diff
@@ -35,7 +36,7 @@ void Hashtable_put(Hashtable* this, ht_key_t key, void* value);
 
 void* Hashtable_remove(Hashtable* this, ht_key_t key);
 
-void* Hashtable_get(Hashtable* this, ht_key_t key);
+void* Hashtable_get(const Hashtable* this, ht_key_t key);
 
 void Hashtable_foreach(Hashtable* this, Hashtable_PairFunction f, void* userData);
```

Comment:
This could likely be made `const` too.

Reply:
Done.

Reply:
Though this may need another look in case any of the `Hashtable_PairFunction` callbacks modify the `Hashtable` since that would lead to UB. If for example: the compiler loads `buckets` and `size` once as an optimization (because `const`), the callback removes elements from the table; it shrinks and `buckets` is freed - this would cause a segfault.